PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.h

```diff
@@ -30,6 +30,7 @@
 
 #include "SizeableDlg.h"
 #include "Common.h"
+#include "localization.h"

```

Comment:
All the header files `*.h` should try to add in source file `.cpp` as possible.

Reply:
done